packages/client/components/TeamPrompt/WorkDrawer/GitHubIntegrationResults.tsx

```diff
@@ -64,6 +64,7 @@ const GitHubIntegrationResults = (props: Props) => {
                 }
               }
             }
+            ...useGetRepoContributions_teamMember
```

Comment:
-1 This seems wrong, we're not using this hook in this component, why do we need to include this fragment then?

Reply:
Removed